data/qmls/point_label.qml

```diff
@@ -1,17 +1,17 @@
-        <text-buffer bufferSize="1" bufferJoinStyle="128" bufferSizeUnits="MM" bufferColor="255,255,255,255" bufferBlendMode="0" bufferSizeMapUnitScale="3x:0,0,0,0,0,0" bufferDraw="0" bufferNoFill="1" bufferOpacity="1"/>
-        <background shapeType="0" shapeSizeType="0" shapeSizeY="0" shapeSizeUnit="MM" shapeOffsetUnit="MM" shapeRotation="0" shapeBorderWidthMapUnitScale="3x:0,0,0,0,0,0" shapeOpacity="1" shapeDraw="0" shapeBorderColor="128,128,128,255" shapeRadiiY="0" shapeSizeX="0" shapeFillColor="255,255,255,255" shapeOffsetY="0" shapeOffsetMapUnitScale="3x:0,0,0,0,0,0" shapeRadiiX="0" shapeBorderWidth="0" shapeRadiiUnit="MM" shapeOffsetX="0" shapeRotationType="0" shapeSVGFile="" shapeJoinStyle="64" shapeSizeMapUnitScale="3x:0,0,0,0,0,0" shapeBlendMode="0" shapeRadiiMapUnitScale="3x:0,0,0,0,0,0" shapeBorderWidthUnit="MM"/>
-        <shadow shadowRadiusAlphaOnly="0" shadowOffsetDist="1" shadowOpacity="0.7" shadowRadius="1.5" shadowBlendMode="6" shadowUnder="0" shadowColor="0,0,0,255" shadowRadiusUnit="MM" shadowScale="100" shadowOffsetMapUnitScale="3x:0,0,0,0,0,0" shadowRadiusMapUnitScale="3x:0,0,0,0,0,0" shadowOffsetUnit="MM" shadowOffsetAngle="135" shadowDraw="0" shadowOffsetGlobal="1"/>
-        <substitutions/>
-      </text-style>
-      <text-format wrapChar="" formatNumbers="0" plussign="0" multilineAlign="3" autoWrapLength="0" useMaxLineLengthForAutoWrap="1" addDirectionSymbol="0" decimals="3" leftDirectionSymbol="&lt;" reverseDirectionSymbol="0" rightDirectionSymbol=">" placeDirectionSymbol="0"/>
-      <placement xOffset="13" repeatDistanceMapUnitScale="3x:0,0,0,0,0,0" distMapUnitScale="3x:0,0,0,0,0,0" centroidInside="0" offsetUnits="MM" repeatDistance="0" yOffset="37" preserveRotation="1" offsetType="0" placementFlags="10" dist="0" maxCurvedCharAngleIn="25" centroidWhole="0" distUnits="MM" maxCurvedCharAngleOut="-25" predefinedPositionOrder="TR,TL,BR,BL,R,L,TSR,BSR" priority="5" placement="0" labelOffsetMapUnitScale="3x:0,0,0,0,0,0" fitInPolygonOnly="0" repeatDistanceUnits="MM" quadOffset="4" rotationAngle="0"/>
-      <rendering drawLabels="1" mergeLines="0" fontMaxPixelSize="10000" limitNumLabels="0" maxNumLabels="2000" upsidedownLabels="0" displayAll="0" obstacleType="0" fontLimitPixelSize="0" zIndex="0" obstacle="1" labelPerPart="0" scaleVisibility="0" fontMinPixelSize="3" minFeatureSize="0" obstacleFactor="1" scaleMin="0" scaleMax="0"/>
-    </settings>
+<qgis>
+  <renderer-v2 type="RuleRenderer">
+    <rules key="0"/>
+    <symbols/>
+  </renderer-v2>
+  <labeling type="rule-based">
+    <rules key="0">
+      <rule key="0" filter="value > 0.1">
+        <settings>
+          <text-style fontSize="10" fontLetterSpacing="0" multilineHeight="1" textColor="0,0,0,255" fontFamily="Sans Serif" fieldName="locality_name"/>
+          <placement predefinedPositionOrder="TR,TL,BR,BL,R,L,TSR,BSR" xOffset="13" yOffset="37" rotationAngle="0"/>
+        </settings>
+      </rule>
+    </rules>
   </labeling>
-</qgis>
+</qgis>
```

Comment:
Minor: Please add newline

Reply:
These files are used in the tests for comparison, which fail if there's a newline, so I'd rather keep them here instead of changing the writer...